ecosystem/typescript/sdk/src/tests/e2e/indexer.test.ts

```diff
@@ -129,6 +129,18 @@ describe("Indexer", () => {
       expect(ledgerInfo.ledger_infos[0].chain_id).toBeGreaterThan(1);
     });
 
+    // OBJECTS //
+
+    it(
+      "gets account owned objects data",
```

Comment:
I think we can test that `alice` has objects since during the test we are creating token and collection, both consider as Objects. so something like that should work

```
        const accountObjects = await indexerClient.getOwnedObjects(alice.address().hex());
        expect(accountObjects.current_objects.length).toBe(2);
      },
```